### ISSUE TITLE
Split <footer> semantically

### ISSUE THREAD

**Issue by nielslange** [WordPress/twentytwenty#564]
As mentioned by @afercia in https://github.com/WordPress/twentytwenty/pull/465/#issuecomment-534580746, the <footer> should be split semantically. 

**Expected behaviour**

* Footer navigation and widget areas belong to `<main>`
* Copyright, powered by and to the top area belong to `<footer>`

**Current behaviour**

* Footer navigation, widget areas, copyright, powered by and to the top area belong to `<footer>`
